components/landing/highlights.json

```diff
@@ -145,7 +145,7 @@
       "title": "",
       "text": "Blocking of Grindr",
       "report": "https://ooni.io/post/iran-internet-censorship/#human-rights-issues",
-      "explore": "/search?input=grindr.com&probe_cc=IR",
+      "explore": "/search?domain=www.grindr.com&probe_cc=IR",
```

Comment:
This query is missing date ranges. cc @agrabeli

Reply:
Added.